src/Compilers/CSharp/csc/Program.cs

```diff
@@ -1,19 +1,30 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
 using System.IO;
 using Microsoft.CodeAnalysis.CommandLine;
-using Roslyn.Utilities;
-using System;
 
 namespace Microsoft.CodeAnalysis.CSharp.CommandLine
 {
     public class Program
     {
         public static int Main(string[] args)
-            => Main(args, Array.Empty<string>());
+        {
+            try
+            {
+                return MainCore(args);
+            }
+            catch (FileNotFoundException e)
+            {
+                // Catch exception from missing compiler assembly.
+                // Report the exception message and terminate the process.
+                Console.Error.WriteLine(e.Message);
```

Comment:
Unless we have a good reason think we should use Console.WriteLine here.  The compiler doesn't use Console.Error at all today.